.github/workflows/main.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           pytest --cov-report=html:html_cov --cov-branch --cov-report term --cov=helical ci/
       
       - name: Upload coverage report
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
```

Comment:
Yeah got the same issue on my branch. Need to make sure this is updated on main afterwards